requirements.txt

```diff
@@ -1,5 +1,34 @@
-flask
-cryptography
-flask-security
-pyjwt
-requests
\ No newline at end of file
+#
```

Comment:
BTW, who is really using the requirements.txt in the library. I think nobody.!

Reply:
If nobody uses then you can get rid of requirements.txt all together.

Reply:
Did not really understand this comment, isn't it required when a developer is working on this repo?

Reply:
We are using below imports in code:

import os
from functools import wraps

so we need requirments.txt right? @arajkumar can you elaborate your comment?

import jwt
import requests
from flask import current_app, request, g

Reply:
It is a library, the usual way of testing library is via unittest right? 
Who ever uses this library will get it's dependency via setup.py(requirement.in).

Reply:
and this library has only two impl files(auth.py, errors.py), rest all are supporting files.